### ISSUE TITLE
Ability to use insertBefore/After when adding new module tab to ProcessPageEdit 

### ISSUE THREAD

**Issue by adrianbj** [ryancramerdesign/ProcessWire#1076]
I am not sure that it's that simple, but in it seems like in modules we are all using:
```
// create the tab
$newTab = new InputfieldWrapper();
$newTab->attr('id', 'newtab');
$newTab->attr('title', 'New Tab');
$newTab->prepend($tabfieldset);
$form->prepend($newTab); // this prepend is putting the $tabfieldset content before the save button
```

The trouble is I think several of us would like to be able to add a new tab between the Content and Children tabs, for example.

Is this method of adding new tabs completely incorrect? Is there any way we can make use of insertBefore / insertAfter ?

**Comment by ryancramerdesign** [comment]
Yes, it's fine to use insertBefore() and insertAfter() with InputfieldWrapper. See /wire/core/InputfieldWrapper.php for the method definitions.

**Comment by adrianbj** [comment]
I might be missing something obvious, but I think the issue is being able to get an existing tab (like "Content") as the $existingItem to use in insertAfter()

$form->prepend($newTab); is prepending the form before the save button. If the position of the new tab was relative to the other tabs, then I would have thought prepend would put it before the Content tab since that is the first part of the form. Note that I am hooking like so:

```
$this->addHookAfter('ProcessPageEdit::buildForm', $this, 'addNewTab');
```
